packages/app/app/containers/TrackPopupContainer/index.js

```diff
@@ -17,6 +17,7 @@ const TrackPopupContainer = props => {
     trigger,
     track,
     artist,
+    t,
```

Comment:
Since this is a container I couldn't use the import { withTranslation } that is used in other places (or at least I don't think so).

So I added the translation-function in the parent component and passed it as a prop to here. Is there a cleaner way of doing it?

Reply:
Why not?

You can import `withTranslation` and `compose` from recompose, and then do
```javascript
export default compose(
  connect(
    mapStateToProps,
    mapDispatchToProps
  ),
  withTranslation('something')
)(TrackPopupContainer);
```

Reply:
I'm still kind of new to React so I haven't learned how everything works yet, so thanks for this suggestion! I'll implement it and then I'll take a look at all translation keys, I've mostly just kebab-cased the actual strings, but most other translation keys are more descriptive of their function. So I'll try to make them more consistent with the other keys!